certified-connectors/Lang-ai/apiProperties.json

```diff
@@ -13,12 +13,31 @@
             "required": "true"
           }
         }
+      },
+      "tenant": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "required": "true"
+          },
+          "description": "Specify your tenant name",
+          "displayName": "Tenant Name",
+          "tooltip": "Provide Your Tenant: (us.app or eu.app or [companyname])"
+        }
       }
     },
     "iconBrandColor": "#3E4EB8",
     "capabilities": [],
-    "publisher": "Lang.ai",
+    "policyTemplateInstances": [
+      {
+        "templateId": "dynamichosturl",
+        "title": "Tenant",
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "https://@connectionParameters('tenant').lang.ai"
+        }
+      }
+    ],
+    "publisher": "Eduardo Vicente",
```

Comment:
@evicentem9 Sorry for late review, but for publisher and stackOwner properties - individuals are not allowed. Can you please provide only company or organization name. Rest everything looks good.

Reply:
@sriyen-msft Thanks for the response. Already changed.